components/list/list-item-generic-layout.js

```diff
@@ -384,6 +416,26 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 		}
 	}
 
+	_onDropdownClose() {
+		if (!this._tooltipShowing) this.frontRow = false;
+		this._dropdownShowing = true;
+	}
+
+	_onDropdownOpen() {
+		this.frontRow = true;
+		this._dropdownShowing = true;
+	}
+
+	_onTooltipHide() {
+		if (!this._dropdownOpen) this.frontRow = false;
```

Comment:
Looks like this `_dropdownOpen` variable should actually be `_dropdownShowing` (based on naming immediately above above).  Whether we care that the variable name matches the event name... 🤷‍♂️ .